tests/grain_size_strain_pinned_state_2/screen-output

```diff
@@ -0,0 +1,868 @@
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.34038e-15, 2.80182e-12, 0.813543
+      Relative nonlinear residual (total system) after nonlinear iteration 3: 0.813543
+
+   Solving temperature system... 0 iterations.
+   Solving grain_size system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.47844e-15, 1.50765e-13, 0.813543
+      Relative nonlinear residual (total system) after nonlinear iteration 4: 0.813543
+
+   Solving temperature system... 0 iterations.
+   Solving grain_size system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.47844e-15, 2.53531e-13, 0.813543
+      Relative nonlinear residual (total system) after nonlinear iteration 5: 0.813543
+
+   Solving temperature system... 0 iterations.
+   Solving grain_size system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.47844e-15, 3.21831e-13, 0.813543
+      Relative nonlinear residual (total system) after nonlinear iteration 6: 0.813543
+
+   Solving temperature system... 0 iterations.
+   Solving grain_size system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.47844e-15, 3.67502e-13, 0.813543
+      Relative nonlinear residual (total system) after nonlinear iteration 7: 0.813543
+
+   Solving temperature system... 0 iterations.
+   Solving grain_size system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.47844e-15, 3.97992e-13, 0.813543
+      Relative nonlinear residual (total system) after nonlinear iteration 8: 0.813543
+
+   Solving temperature system... 0 iterations.
+   Solving grain_size system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.47844e-15, 4.18131e-13, 0.813543
+      Relative nonlinear residual (total system) after nonlinear iteration 9: 0.813543
+
+   Solving temperature system... 0 iterations.
+   Solving grain_size system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 1+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 1.47844e-15, 4.31876e-13, 0.813543
```

Comment:
I guess the Stokes residual still does not decrease. That's still something we need to figure out.